### DISCUSSION TITLE
Do you guys have any plan to bring back the lagacy terminal functionalities to future releases ?

### DISCUSSION THREAD

**Discussion by anik2k06** [wavetermdev/waveterm#989] [General]
### Feature description

Do you guys have any plan to bring back the lagacy terminal functionalities to future releases or it will be that same old plain terminal as before ?
I mean the chat like ui as before . a text box at the bottom and each command as a block.
Or it will be like the plain terminal.
will you bring back workspaces?

**Comment by sawka** [comment]
workspaces will definitely be coming back.  mainly working on new UI to expose them.

as for the chat-like block interface for the terminal, that's harder.  we've had a lot of internal discussions about how to bring that back as an optional interface.  those discussions are still on-going.  but if it were to come back it would not be in the short term.  if it does come back, probably not until sometime next year.
